ComplementaryScripts/Functions/importHumanYaml.m

```diff
@@ -0,0 +1,307 @@
+function model=importHumanYaml(yamlFilename)
+% importHumanYaml
+%   Imports a yaml file matching (roughly) the cobrapy yaml structure
+%
+%   Input:
+%   yamlFile    a file in yaml model structure. As defined in HumanGEM, the
+%               yaml file contains 5 sections: metaData, metabolites,
+%               reactions, genes and compartments
+%
+%   Output:
+%   model       a model structure
+%
+%   Usage: model=importYaml(yamlFilename)
+%
+%   Hao Wang, 2019-07-26
+%
+% This function is to reverse engineer the RAVEN function `writeYaml`
+%
+
+if ~(exist(yamlFilename,'file')==2)
+    error('Yaml file %s cannot be found',string(yamlFilename));
+end
+
+% Define the required fields of humanGEM
```

Comment:
I'd consider doing this assignment on a single line.

Reply:
sorry I don't understand.

Reply:
Something like this:
`[model.id, model.description] = deal([], [])`
`[model.rxn, model.mets] = deal({}, {}) `